pkg/api/v1/schema.go

```diff
@@ -35,8 +35,20 @@ type DomainSpec struct {
 }
 
 type Memory struct {
-	Value uint   `json:"value"`
-	Unit  string `json:"unit"`
+	// Allocated field defines the amount of memory a VM gets upon starting in kiloBytes
+	// The field type means the total maximum is 4 TiB of memory
+	Allocated uint `json:"allocated"`
+
+	// Required fields defines the amount of memory a VM needs to operate properly.
+	// This field can be set to the same value as `Allocated` to effectively
+	// disable any memory overcommit features. The value is provided in Bytes.
```

Comment:
it's weird that this is represented in bytes when the other values in this struct are kilobytes. 

using k8s style of representing memory unit and value in the same string clears this up too.

Reply:
That is a typo.. it was supposed to be in KiB.